models.py

```diff
@@ -29,40 +35,54 @@ def generate_compiled_segmentation_model(model_name, model_parameters, num_class
 
     crossentropy = BinaryCrossentropyL() if num_classes == 1 else CategoricalCrossentropyL()
     loss_fn = crossentropy
+
     all_metrics = []    # one-hot versions are generally preferred for given metric
+    # make first metric a copy of loss, to verify `val_loss` is correct
     if isinstance(loss_fn, BinaryCrossentropyL):
-        all_metrics.append(BinaryCrossentropyM())
+        all_metrics.append(BinaryCrossentropyM(name='binary_ce_metric'))
     else:
-        all_metrics.append(CategoricalCrossentropyM())
-    all_metrics[0].name = str(all_metrics[0].name + '_keras_metric')
+        all_metrics.append(CategoricalCrossentropyM(name='categ_ce_metric'))
 
     for class_num in range(num_classes + 1):
         if class_num == 0:    # all class metrics
             # note, `loss_fn` for all classes placed before `all_metrics` in lineup of command window metrics and plots
             all_metrics.extend([
-                                CategoricalCELoss(),
-                                Accuracy(),
-                                OneHotAccuracy(),
-                                OneHotClassBinaryAccuracyKeras(),
-                                OneHotClassBinaryAccuracySM(),
-                                CategoricalAccuracy(),
-                                FalseNegatives(name='false_neg', thresholds=global_threshold),
-                                OneHotFalseNegatives(name='false_neg_1H'),
-                                TrueNegatives(name='true_neg', thresholds=global_threshold),
-                                OneHotTrueNegatives(name='true_neg_1H'),
-                                FalsePositives(name='false_pos', thresholds=global_threshold),
-                                OneHotFalsePositives(name='false_pos_1H'),
-                                TruePositives(name='true_pos', thresholds=global_threshold),
-                                OneHotTruePositives(name='true_pos_1H'),
-                                Recall(name='recall', thresholds=global_threshold),
-                                OneHotRecall(name='recall_1H'),
-                                Precision(name='precision', thresholds=global_threshold),
-                                OneHotPrecision(name='precision_1H'),
-                                FBetaScore(name='f1_score', beta=1, thresholds=global_threshold),
-                                OneHotFBetaScore(name='f1_score_1H', beta=1),
-                                IoUScore(name='iou_score', thresholds=global_threshold),
-                                OneHotIoUScore(name='iou_score_1H')
-                                ])
+                CategoricalCELoss(),
+                AccuracyKeras(name='accuracy_keras'),
+                OneHotAccuracyKeras(),
+                AccuracyTfKeras(name='accuracy_tfkeras'),
+                OneHotAccuracyTfKeras(),
+                # OneHotHotAccuracyTfKeras(),       # test location of 1H implementation
```

Comment:
similar comment to the one I made elsewhere - we should either remove these lines or add a verbose flag (or, actually, just make a test might be a good move here)

Reply:
removed, only for debugging during version upgrade